include/emp/prefab/ValueBox.hpp

```diff
@@ -90,6 +90,53 @@ namespace emp::prefab {
     }
   };
 
+  /**
+   * Use a LiveValueDisplay component to display a labeled value which will
+   * change over the course of a simulation with a nice description of what
+   * this value means. Internally, it manages an Animator to redraw the
+   * view at the fastest possible framerate unless independent=false.
+   *
+   * We suggest adding the "display_group" class to the enclosing tag around
+   * multiple LiveValueDisplays to align labels and values along a common grid.
+   */
+  class LiveValueDisplay : public ValueBox {
+
+    public:
+    /**
+     * @param label name for this value
+     * @param desc a more detailed description of what the value means
+     * @param value a value to display or function returning a string
+     * @param independent should this component be independently animated? Or will
+     * some other component manage its animation?
+     * @param id user defined ID for LiveValueDisplay div (default is emscripten generated)
+     */
+    template<typename T>
+    LiveValueDisplay(
+      const std::string & label,
+      const std::string & desc,
+      T && value,
+      const bool & independent=true,
+      const std::string & id=""
+    ) : ValueBox(label, desc, id) {
+      view << web::Live(std::forward<T>(value));
+      if (independent) {
+        this->AddAnimation(GetID(), [](){;}, view);
+        this->Animate(GetID()).Start();
+      }
+    }
+
+    /**
+     * Returns the view div containing live values to animate
+     * (use this in an Animate instance).
+     *
+     * @return the view div
+     */
+    Div & GetView() {
+      return view;
```

Comment:
Is view a member of the parent class? If so, it would make more sense to me to put the accessor there.